### PR TITLE
Give ability to pass the list of properties to show as parameter

### PR DESCRIPTION
**Pull request by lorenzofox3**
Set the list of properties to show as a parameter

### CODE REVIEW

**Comment by jonathanong** [comment]
there are too many unnecessary changes here. i will see about reimplementing it myself

**Comment by lorenzofox3** [comment]
Up to you. But at the end there is only one line which changes  https://github.com/lorenzofox3/json-error/blob/master/index.js#L10 (plus a test added). The rest is just code style issues raised by linting tool
like missing semi-colon, missing curly braces for if statements, etc.

Thanks anyway for the review.

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 .DS_Store*
 node_modules
 coverage
+.idea
```
Comment (wilmoore) [inline .gitignore:4]: I think IDE specific ignores should go into your global git ignore file.

File: index.js [hunk 2]
```diff
@@ -1,41 +1,46 @@
...
+];
+
+module.exports = function (propertiesToShow) {
+
+  var props = propertiesToShow || defaultProps;
 
-module.exports = function () {
   return function* jsonErrorHandler(next) {
-    var status
+    var status;
```
Comment (wilmoore) [inline index.js:13]: Probably shouldn't add a semi-colon unless you are going to add them everywhere; however, I think that is counter to this particular project's style.

**Pull request closed without merging**